components/Card/Card.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactNode } from 'react';
+import { cnb } from 'cnbuilder';
+import * as types from './Card.types';
+import * as styles from './Card.styles';
+
+export interface CardProps extends React.HTMLAttributes<HTMLElement> {
+  as?: types.CardType;
+  className?: string;
+  children?: ReactNode;
+  icon?: styles.IconType;
+  // Title for the SVG for accessibility
+  title?: string;
```

Comment:
Same for here, since I imagine you wouldn't add title for some icons in this card, and no title for other icons